src/main/controlMod.F90

```diff
@@ -1101,6 +1105,8 @@ subroutine control_print ()
        write(iulog, *) '    use_fates_fixed_biogeog = ', use_fates_fixed_biogeog
        write(iulog, *) '    use_fates_nocomp = ', use_fates_nocomp
        write(iulog, *) '    use_fates_sp = ', use_fates_sp
+       write(iulog, *) '    use_fates_luh= ', use_fates_sp
+       write(iulog, *) '    fluh_timeseries = ',fluh_timeseries
```

Comment:
Add a trim around character strings...

Reply:
addressed in b06181632f3e7609d6a71fd3b55fa0b4fe5fc800